elizhahn/static-comp
# Feature/header

## Is this a fix or a feature?
- feature

## What is the change?
- a header for the application site was added and includes:
     -  navigational buttons
     - logo with logo image
     - admin button
   
- cross browser compatibility is included, and responsive design for tablet and mobile screen sizes

## What does it fix?

## Where should the reviewer start?

- index.html
- styles.css

## How should this be tested?
- open index.html and view application on chrome, firefox, and safari, each view should render the same
- in addition change the screen size to under 850, the screen layout should change in an appealing way and change the screen size to under 550, the layout should adjust again for phone sized screens in an appealing way. 

## Screenshots (if appropriate):
